src/status_im2/contexts/wallet/account/bridge/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im2.contexts.wallet.account.bridge.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im2.contexts.wallet.account.bridge.style :as style]
+    [status-im2.contexts.wallet.common.account-options.view :as account-options]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
```

Comment:
I think we should wrap this view with the theme HOC unless this page doesn't use the theme key.

Reply:
It doesn't use `theme`